Added adult keywords

robynk opened the pull request.

robynk committed:
Added adult keywords

kane-c commented:
Same here - need commas at the end of those lines.

kane-c commented on src/pages/personalisation/Age.js:
```diff
@@ -24,16 +24,19 @@ export default class Age extends BaseQuestion {
                 ],
             }),
             "55 to 64": append({
+                q: "(pre-retirement aged"
```
Missing closing `)`

kane-c commented:
Also as far as I can tell, the query param `age_groups` should actually be `age_group`

robynk committed:
Fixed mistakes

Added commas at end of lines, changed age_groups to age_group and closed brackets

DanielHeath committed:
Merge pull request #367 from ask-izzy/robynk-patch-3-1

Fixed mistakes

DanielHeath committed:
Fixup

DanielHeath commented:
Merged into https://github.com/ask-izzy/ask-izzy/pull/365
